src/pipeline/src/manager/table.rs

```diff
@@ -57,6 +57,11 @@ pub type PipelineVersion = Option<TimestampNanosecond>;
 
 pub type PipelineTableRef = Arc<PipelineTable>;
 
+pub type PipelineRef = Arc<Pipeline<GreptimeTransformer>>;
+
+/// Pipeline info. A tuple of timestamp and pipeline reference.
+pub type PipelineInfo = (Timestamp, PipelineRef);
```

Comment:
`PipelineRef` is not actually used, Can we just return `PipelineVersion`?

Reply:
A detailed description of the pipeline should be returned in the future.